.github/workflows/provider-compatibility.yml

```diff
@@ -10,9 +10,9 @@
 
 name: Provider compatibility across versions
 
-on: #[pull_request]
-  schedule:
-    - cron: '0 15 * * *'
+on: [pull_request]
```

Comment:
Please remove this change after the tests all pass.

Reply:
The commit is tagged as to be deleted.  So definitely.